opennebula/defaults.yaml

```diff
@@ -85,13 +66,7 @@ kernel:
     novnc:
       service: {}
     oneflow:
-      pkgs:
-        - opennebula-flow
-        - ruby-treetop
-        - ruby-polyglot
     onegate:
```

Comment:
Please remove this line since onegate becomes a None-typed variable that can be merged with other dicts.